src/General/Alert/Alert.tsx

```diff
@@ -98,36 +79,37 @@ export const Alert = ({
     }
   }, [type]);
 
-  return isVisible ? (
-    <AlertContainer
-      className={classNames('aries-alert', className)}
-      type={type}
-      role="alertdialog"
-      aria-hidden={isVisible ? 'false' : 'true'}
-      aria-describedby="alert-message"
-      isOpen={isOpen}
-      isVisible={isVisible && isOpen}
-      tabIndex={0}
-      onKeyDown={handleKeyDown(onClose)}
-      ref={alertContainerRef}
-    >
-      <AlertTypeIconContainer>
-        <AlertTypeIcon color={alertColor} />
-      </AlertTypeIconContainer>
-      <AlertContent className="alert-content">
-        <AlertMessage className="alert-message">{message}</AlertMessage>
+  return (
+    <DelayedUnmount mount={isVisible} delay={290}>
```

Comment:
Decreased to 290ms. The close animation duration is 300ms and if the unmount delay is equal to that, the alert will "flicker" back to its open position briefly before unmounting.